app/models/generated_report.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "defra_ruby/exporters"
+
+class GeneratedReport < ActiveRecord::Base
```

Comment:
Will it be a pain to move this to the `reports` namespace? 😁

Reply:
Oh yeah! 👍